### PR TITLE
Update Mini-Cart block editor sidebar

### PR DESCRIPTION
**Pull request by Aljullu**
This PR:

* Updates the design of the Mini-Cart block sidebar in the editor.
* Replaces all instances of "Mini Cart" with "Mini-Cart", according to the discussion here: pdnLyh-3ze-p2#comment-2425.

#### User Facing Testing

1. Add the Mini-Cart block to the header of your site.
2. Click on the block and open the editor sidebar.
3. Verify the strings from the design below match the UI (note: it's expected that some options from the design are still not implemented, just focus on the strings of the options that exist in both screenshots):

Design | Actual
--- | ---
imatge | imatge

4. Verify there are no longer mentions to "Mini Cart", instead all strings mention now "Mini-Cart".

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Update Mini-Cart block editor sidebar with a clearer design and texts

### CODE REVIEW

File: assets/js/blocks/mini-cart/edit.tsx [hunk 1]
```diff
@@ -136,27 +108,62 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
...
+					{ templatePartEditUri && (
+						<>
+							<img
+								className="wc-block-editor-mini-cart__drawer-image"
+								src="/wp-content/plugins/woocommerce-blocks/images/blocks/mini-cart/cart-drawer.svg"
+								alt=""
+							/>
+							<p>
+								{ __(
+									'When opened, the mini-cart drawer gives shoppers quick access to view their selected products and checkout.',
```
Comment (albarin) [inline assets/js/blocks/mini-cart/edit.tsx]: Just a doubt, is there any reason why this `mini-cart` is not capitalized?

File: assets/js/blocks/mini-cart/mini-cart-contents/index.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const settings: BlockConfiguration = {
 	category: 'woocommerce',
 	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
 	description: __(
-		'Display a mini cart widget.',
+		'Display a mini-cart widget.',
```
Comment (albarin) [inline assets/js/blocks/mini-cart/mini-cart-contents/index.tsx]: Same question here.

**Review by albarin — APPROVED** [review]
I left a question, but looks great, approving 👌

**Comment by Aljullu** [comment]
Thanks for the review, @albarin! I have fixed both instances in 58c74773234c1ebd51a4b14c7d955ef873d2eca4.

**Pull request merged**